docs/components/zeebe/technical-concepts/partitions.md

```diff
@@ -48,6 +48,20 @@ A partition is a persistent append-only event stream. Initially, a partition is
 
 For fault tolerance, data in a partition is replicated from the **leader** of the partition to its **followers**. Followers are other Zeebe broker nodes that maintain a copy of the partition without performing event processing.
 
+We recommend an odd replication factor. Even replication factors can be problematic and may trigger this warning:
+
+```
+ "Expected to have odd replication factor, but was even ({}). Even replication factor has no benefit over "
+          + "the previous odd value and is weaker than next odd. Quorum is calculated as:"
+          + " quorum = floor(replication factor / 2) + 1. In this current case the quorum will be"
+          + " quorum = {}. If you want to ensure high fault-tolerance and availability,"
+          + " make sure to use an odd replication factor.";
+```
```

Comment:
```suggestion
We recommend an odd replication factor. 
```

Reply:
I would just remove this, since we explain it later anyway.

Wdyt?

Reply:
I included it in case someone copies and pastes it into external or internal search. It is a bit redundant.

We can remove it for now and add it in the future if we want to leverage the SEO benefits.